grpc/service_config/service_config.proto

```diff
@@ -527,6 +527,15 @@ message LeastRequestLocalityLoadBalancingPolicyConfig {
   uint64 choice_count = 1;
 }
 
+
+// Configuration for the override_host LB policy.
+message OverrideHostLoadBalancingPolicyConfig {
+  // override_host_status in CDS includes values we support for stateful session affinity.
```

Comment:
This comment should just say something like "if false, the LB policy will be a no-op."  It does not need to reference anything xDS-specific.

Reply:
done